### PR TITLE
Adds torpedo juice - Attempt number 2

### PR DESCRIPTION
**Pull request by SharkeeByond**
This time I DID NOT fuck up the filepaths. Anyway, credit to God, once again.
## About The Pull Request

Adds Torpedo Juice and a new trait - ''Munitions Metabolism''. Torpedo Juice heals people with this trait.

## Why It's Good For The Game

Bolsters the variety of alcoholic drinks, and it also adds a unique NSV-themed drink, plus a new metabolism trait which could
be used for other things.

## Changelog
:cl:
add: Torpedo juice 
add: Munitions metabolism trait
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
Really neat idea, I'm happy to go with this.

Consider making variants for other weapons? (Mac spam juice?)

**Comment by Pontenerd** [comment]
> Really neat idea, I'm happy to go with this.
> 
> Consider making variants for other weapons? (Mac spam juice?)

Hello I am spriter man, and I am when I can be bothered to do so.

File: code/__DEFINES/traits.dm [hunk 1]
```diff
@@ -147,6 +147,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define	TRAIT_CALCIUM_HEALER	"calcium_healer"
 #define	TRAIT_MAGIC_CHOKE		"magic_choke"
 #define TRAIT_SOOTHED_THROAT    "soothed-throat"
+#define TRAIT_MUNITIONS_METABOLISM    "munitions-metabolism"
```
Comment (Karmic-Skink) [inline code/__DEFINES/traits.dm:150]: //NSV13 tag - Also did you check the global vars?

**Comment by Karmic-Skink** [comment]
Don't really like the idea of just being in a particular department giving you access to additional forms of healing.

**Comment by SharkeeByond** [comment]
Closing this pull request because I accidentally orphan-ed it.

**Pull request closed without merging**